colour/models/rgb/ycbcr.py

```diff
@@ -0,0 +1,359 @@
+    'legal_12_int' : [256, 3760]
+    'legal_12_YC_int' : [256, 3760, 256, 3840]
+    'legal_10' : [64./1023, 940./1023]
+    'legal_10_YC' : [64./1023, 940./1023, 64./1023, 960./1023]
+    'legal_10_int' : [64, 940]
+    'legal_10_YC_int' : [64, 940, 64, 960]
+    'legal_8' : [16./255, 235./255]
+    'legal_8_YC' : [16./255, 235./255, 16./255, 240./255]
+    'legal_8_int' : [16, 235]
+    'legal_8_YC_int' : [16, 235, 16, 240]
+    'full' : [0.0, 1.0]
+    'YPbPr' : [0.0, 1.0, -0.5, 0.5]
+"""
+
+
+def rgb_to_YCbCr(rgb, K=WEIGHT['Rec.709'], inRange=RANGE['full'],
+                 outRange=RANGE['legal_10_YC']):
+    """
+    Converts an array of R'G'B' values to the corresponding Y'CbCr values.
+
+    Note: For ITU-R BT.2020 (Rec.2020) the rgb_to_YCbCr function is only
+    applicable tothe non-constant luminance implementation.
+    The rgb_to_YcCbcCrc function should be used for the constant luminance case
+    See https://www.itu.int/dms_pubrec/itu-r/rec/bt/\
+    R-REC-BT.2020-0-201208-S!!PDF-E.pdf
+
+    Parameters
+    ----------
+    rgb : array_like
+        Input R'G'B' values. These may be floats or integers.
+    K : array of two values
+        The luma weighting coefficients of red and blue.
+        Default: WEIGHT['Rec.709']
+    inRange : array of two values (optional)
+        The values of R', G' and B' corresponding to 0 IRE and 100 IRE.
+        These may be floats or integer code values.
+        Default: RANGE['full']
+    outRange : array of four values (optional)
+        The values of Y' corresponding to 0 IRE and 100 IRE
+        and the minimum and maximum values of Cb and Cr.
+        These may be floats or integer code values.
+        Default: RANGE['legal_10_YC']
+
+    Returns
+    -------
+    ndarray
+        Y'CbCr array.
+        These may be floats or integer code values, depending on the parameter
+        values given for outRange.
+
+    Examples
+    --------
+    >>> RGB = np.array([1.0, 1.0, 1.0])
+    >>> rgb_to_YCbCr(RGB)  # doctest: +ELLIPSIS
+    array([ 0.91886608,  0.50048876,  0.50048876])
+    >>> rgb_to_YCbCr(RGB, outRange=RANGE['legal_10_YC_int'])
+    array([940, 512, 512])
+
+    For the JFIF JPEG conversion shown on https://en.wikipedia.org/wiki/YCbCr:
+
+    >>> RGB = np.array([102, 0, 51])
+    >>> rgb_to_YCbCr(RGB,
+    ...              K=WEIGHT['Rec.601'],
+    ...              inRange=[0, 255],
+    ...              outRange=[0, 255, 0, 256])  # doctest: +ELLIPSIS
+    array([ 36, 136, 175])
+
+    Note the use of 256 for the max Cb/Cr value, which is required so that the
+    Cb and Cr output is centred about 128. Using 255 would centre them about
+    127.5 meaning that there was no integer code value to represent neutral
+    colours. This does however create the possibility of output values of 256,
+    which cannot be stored in an 8-bit integer.
+    """
+    Kr = K[0]
+    Kb = K[1]
+    intOut = (isinstance(outRange[0], int) and isinstance(outRange[1], int) and
```

Comment:
1. `intOut` ---> `int_out` or `int_o` (underscore case) :)
2. Could do something along that line maybe: `int_o` = all([isinstance(x, int) for x in out_range])`